packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -16,6 +16,10 @@ const propTypes = {
    * The text content that specifies an alternative text for an image.
    */
   alt: PropTypes.string,
+  /**
+  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
```

Comment:
Let's add an additional warning about people photos and which props are recommended they use (and `fill` should never be used since it squishes a face out of proportion):

```suggestion
  * Sets the `object-fit` style of the image from the following values; `cover`, `contain`, `fill`, `scale-down`, `none`.
  * ![IMPORTANT](https://badgen.net/badge//IMPORTANT/CSS?icon=github) Anywhere the terra-profile-image is used to show images of PEOPLE, _only_ `cover` and `contain` are acceptable, `fill` should _never_ be used.
```

Reply:
Updated the warning https://github.com/cerner/terra-core/pull/2804/commits/c2ab5d24867dca1fd16b710775b4557f9ebcdeee, since i have removed the `fill` value from the list of possible values of fit.
Explanation here : https://github.com/cerner/terra-core/pull/2804#issuecomment-592416707